spec/mailers/previews/ticket_preview.rb

```diff
@@ -5,4 +5,9 @@ def new_ticket
     ticket = Ticket.last
     TicketMailer.new_ticket(user:, ticket_id: ticket.id)
   end
+
+  def missing_ticket
+    user = User.last
```

Comment:
Just wondering why you search for the last user/last ticket in this file. Are you sure this won't be an issue and that it may not get the wrong data?

Reply:
Yeap. Those things could happen. We are in dev mode, so, we need a record to see the email preview.
https://github.com/rootstrap/poc-action-mailbox/pull/6#discussion_r1086989410.  Same here